apps/src/lib/levelbuilder/rubrics/RubricsContainer.jsx

```diff
@@ -125,28 +195,42 @@ export default function RubricsContainer({
     })
       .then(response => response.json())
       .then(data => {
-        navigateToHref(data.redirectUrl);
+        if (!rubric) {
```

Comment:
This was just the start of working on making the save button work for editing a rubric.   This doesn't hurt the saving of new rubrics, but is a change from the original code.  I decided to leave it in because I am working on the edit piece next.